index.py

```diff
@@ -682,8 +682,27 @@ def deg2num(lat_deg, lon_deg, zoom):
 
 
 if __name__ == "__main__":
-    web.wsgi.runwsgi = lambda func, addr=None: web.wsgi.runfcgi(func, addr)
```

Comment:
WSGI is how app works in production. why is it stripped?

Reply:
The `__name__ == "__main__"` part should not be invoked when the script is imported as WSGI application. This part only works if the script is directly run with Python. For example, for local development as `python index.py`.

But I am not sure what happens if the server is configured as FastCGI, and not WSGI.